### PR TITLE
Webpack implementation

### PR DESCRIPTION
**Pull request by RicardoEgas**
In this branch, I implemented the following;

- Set up Webpack config file and bundled the js file;
- Added an object with different tasks and sorted the array by index;
- Added some styling to match the original design;

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  },
+];
+
+const checkTasks = () => {
+  const lists = document.querySelector('.lists');
+
+  tasks.sort((a, b) => a.index - b.index);
+
+  tasks.forEach((task) => {
+    lists.insertAdjacentHTML('beforeend', `<li><input type="checkbox">${task.description}<i class="fa-solid fa-ellipsis-vertical"></i></li>`);
```
Comment (DuaneDave) [inline src/index.js:32]: Note, according to the [wireframe ](https://www.youtube.com/watch?v=AcUd-_Yjjqg)provided, the container for the Todo shouldn't be taking the full width of the screen as yours is currently doing. Consider resolving this at least to make your design similar to the provided template.

Your snapshot
see

**Review by DuaneDave — CHANGES_REQUESTED** [review]
Hi @RicardoEgas  👋 , 

Good job so far on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Correct set up of webpack
✔️ The function to create a Todo is working as expected

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by anagudelogu — APPROVED** [review]
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi @RicardoEgas, 
I'm Andres.

You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 



## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Great UI design. 💻 
✅ Descriptive PR title, summary and commit messages. 💬 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Cabraham1 — COMMENTED** [review]
Hi @RicardoEgas , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**